doc/locale/ja/LC_MESSAGES/reference/commands/select.po

```diff
@@ -3185,6 +3186,41 @@ msgstr ""
 "ません。この場合は ``_key`` を使ってください。これは、 :ref:`select-"
 "drilldown-output-columns` と同じルールです。"
 
+msgid "Specify ``${LABLE}`` of other ``drilldowns`` or ``slices``."
+msgstr "他の ``drilldowns`` または ``slices`` の ``${LABLE}`` を指定します。"
+
+msgid ""
+"You can drilldown the result of specified ``${LABLE}``. It means that this "
+"parameter enables a nested drilldown."
+msgstr ""
+"指定した ``${LABLE}`` の結果をドリルダウンします。つまり、このパラメータを使"
+"うと多段ドリルダウンができます。"
+
+msgid ""
+"Here is an example to execute the nested drilldown by tag and then by "
+"category."
+msgstr "以下は、タグ、カテゴリの順で多段ドリルダウンを行う例です。"
+
+msgid ""
+"In this example; The schema contains the table named as ``Memo`` which has "
+"the column named as ``tag``, the table named as ``Tags`` which has the "
+"columns named as ``category``."
```

Comment:
```suggestion
"column named as ``category``."
```
isn't the column named "category" is a single column?

Reply:
yes, it is a single column.
I will fix it.

Reply:
I have fixed it.